SwiftWisdomTests/Rx/OperatorTests.swift

```diff
@@ -8,16 +8,17 @@
 
 import XCTest
 import RxSwift
+import RxCocoa
 @testable import SwiftWisdom
 
 
 class OperatorTests: XCTestCase {
 
     func testAddCompositeDisposable() {
         let compositeDisposable = CompositeDisposable()
-        let observable = Observable<String>.never()
+        let observable = Observable<String?>.never()
         XCTAssertEqual(0, compositeDisposable.count)
-        UILabel().rx_text <- observable >>> compositeDisposable
+        let _ = UILabel().rx.text <- observable >>> compositeDisposable
```

Comment:
question here - does this mean that everytime we use >>> we would need to do let _ ? that doesn't seem right?